administrator/components/com_media/libraries/media/file/adapter/local.php

```diff
@@ -280,10 +280,16 @@ private function getPathInformation($path)
 
 		try
 		{
-			// Get the image properties
-			$props       = JImage::getImageFileProperties($path);
-			$obj->width  = $props->width;
-			$obj->height = $props->height;
+			$obj->width  = 0;
+			$obj->height = 0;
+
+			if (!is_dir($path))
+			{
+				// Get the image properties
+				$props       = JImage::getImageFileProperties($path);
+				$obj->width  = $props->width;
+				$obj->height = $props->height;
+			}
```

Comment:
What happens in case of non-image files like html, mp4, pdf? Perhaps we need an isImage() check.